velox/tpch/gen/dbgen/build.cpp

```diff
@@ -85,11 +87,13 @@ static void gen_phone(DSS_HUGE ind, char* target, seed_t* seed) {
   RANDOM(exchg, 100, 999, seed);
   RANDOM(number, 1000, 9999, seed);
 
-  sprintf(target, "%02d", static_cast<int>(10 + (ind % NATIONS_MAX)));
-  sprintf(target + 3, "%03d", static_cast<int>(acode));
-  sprintf(target + 7, "%03d", static_cast<int>(exchg));
-  sprintf(target + 11, "%04d", static_cast<int>(number));
-  target[2] = target[6] = target[10] = '-';
+  std::string phone = fmt::format(
+      "{}-{:03d}-{:03d}-{:04d}",
```

Comment:
Should it start with `{:02d}`?
Do we have existing tests for this?

Reply:
The range of the expression :  10 + (ind % NATIONS_MAX) is [10,99] since NATIONS_MAX = 90, so it will always be a 2 digit number. That's why the expression works. `TpchGenTestSupplierTest` contains  a unit test named `batches` which checks this. Still to be on a safer side I will add {:02d} .